cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.4
+
+- adding an input list for a mix of dimensions
+- validate each input from the user input list, if at least one exists, it runs anomaly report
```

Comment:
Policy seems to fail if at least one dimension passed is invalid.

Reply:
Yes, ill add specific description